language/move-prover/spec-lang/src/ast.rs

```diff
@@ -248,6 +248,7 @@ pub enum Exp {
     Lambda(NodeId, Vec<LocalVarDecl>, Box<Exp>),
     Block(NodeId, Vec<LocalVarDecl>, Box<Exp>),
     IfElse(NodeId, Box<Exp>, Box<Exp>, Box<Exp>),
+    Empty(NodeId),
```

Comment:
Instead of introducing a new expression node, could you introduce instead Operation:::Empty and represent this as Call(node_id, Empty, [])? This is the current design philosophy here. The number of Exp variants should be kept to the minimum. This would also prevent the need to extend various matches here and there.